DotNetExercise/Exercise03.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotNetExercise
+{
+    public class Exercise03
+    {
+        private static bool isPrimeNumber(int n)
+        {
+            if (n < 2)
+            {
+                return false;
+            }
+            int squareRoot = (int)Math.Sqrt(n);
+            for (int i = 2; i <= squareRoot; i++)
+            {
+                if (n % i == 0)
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+
+        public static string GetProductOfPrimeNumbers(int number)
+        {
+            int i = 2;
+            string productOfPrimeNumbers = "";
+
+            while (number > 1)
+            {
+                if (isPrimeNumber(i) && number % i == 0)
```

Comment:
Cái vế sau có vẻ không cần lắm, em check lại xem


```suggestion
                if (isPrimeNumber(i))
```

Reply:
Ok anh.